index.html

```diff
@@ -30,7 +31,77 @@
       <p id="connect">LET’S CONNECT</p>
       <img id="icons" src="assets/images/social-media-icons.svg" alt="social-media icons">
   </section>
-    
-    
+</div>
+   
+
+  <div id="section1" class="section_2">
+    <img id="snap" src="assets/images/Snapshoot_Portfolio.svg" alt="Snapshot of portfolio">
+    <img id="tonic" src="assets/images/tonic.png" alt="Tonic Project name">
```

Comment:
- [ ] the title of the card should be a text and not an image,  kindly remove the image and add `<hx>` as title, and the list below it should be `<ul>` with `<li>` items